CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Next
```

Comment:
If we're sticking to http://keepachangelog.com/ , then this should probably be `## Unreleased` or `## [Unreleased]`